pages/index.js

```diff
@@ -1,3 +1,8 @@
+// Modified this file to restyle form
+// I'm really sorry. When I made these commits, I was on a different branch on a different PR
+// The changes I made to this file was to re-style the two form components and the home page
+// I'll be more careful next time. I wasn't thinking
+
```

Comment:
lol all good, make sure to delete this comment before merging. (Next time you can make comments like these in the Pull request/through Github comments, rather than committing them in code)